ci-operator/step-registry/multiarch/validate-nodes/multiarch-validate-nodes-commands.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ -z "${CONTROL_ARCH}" ] || [ -z "${COMPUTE_ARCH}" ]; then
+  echo "[WARN] Skipping this test as either CONTROL_ARCH or COMPUTE_ARCH are not set."
+  exit 0
+fi
+
+if ! command -v yq-v4 &> /dev/null; then
```

Comment:
What case need to download the `yq-v4`, I saw the step is using `upi-installer` image to run.

Reply:
We'd need to backport the patch that included yq-v4 to older OCP versions still maintenance

Reply:
I did not see the step has dependency on openshift-install binary, the step just want to use `yq`, if I am right, we can use a specific tag of upi-installer image. e.g:
```
  from_image:
    namespace: ocp
    name: "4.14"
    tag: upi-installer
```

Once that, we do not need to the download step for `yq`.

Reply:
done